### PR TITLE
Lists what hostnames and IPs are needed for custom certificates

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1454482

### CODE REVIEW

File: install_config/registry/deploy_registry_existing_clusters.adoc [hunk 1]
```diff
@@ -380,6 +380,34 @@ MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCyOJ5garOYw0sm
 ...
 -----END PRIVATE KEY-----
 ----
+
+- If using Subject Alternative Names (SAN), the server certificate should contain the following in the SAN list:
```
Comment (rjhowe) [inline install_config/registry/deploy_registry_existing_clusters.adoc]: Remove "If using Subject Alternative Names (SAN)". 

The secured registry should contain the following in the SAN list

File: install_config/registry/deploy_registry_existing_clusters.adoc [hunk 2]
```diff
@@ -380,6 +380,34 @@ MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCyOJ5garOYw0sm
 ...
 -----END PRIVATE KEY-----
 ----
+
+- The secured registry should contain the following Subject Alternative Names (SAN) list:
++
+* Two service nostnames:
```
Comment (zhaozhanqi) [inline install_config/registry/deploy_registry_existing_clusters.adoc]: s/nostnames/hostnames

**Comment by mburke5678** [comment]
@zhaozhanqi @rjhowe What OpenShift versions does this affect? The customer case involves 3.4, Should this PR be merged 3.4+?

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: install_config/registry/deploy_registry_existing_clusters.adoc [hunk 3]
```diff
@@ -380,6 +380,34 @@ MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCyOJ5garOYw0sm
...
+- The secured registry should contain the following Subject Alternative Names (SAN) list:
++
+* Two service hostnames:
++
+----
+docker-registry.default.svc.cluster.local
+docker-registry.default.svc 
+----
++
+* service IP address. Use the following command to get the Docker registry service IP address:
```
Comment (bfallonf) [inline install_config/registry/deploy_registry_existing_clusters.adoc]: Looks like there's examples above, but this isn't showing an example. It's showing how to find what you need to put into the file. Is that correct? If yes, I'd suggest consistency, whether through how to find the example or the example itself. This goes for the below as well.

File: install_config/registry/deploy_registry_existing_clusters.adoc [hunk 4]
```diff
@@ -380,6 +380,34 @@ MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCyOJ5garOYw0sm
...
++
```
Comment (bfallonf) [inline install_config/registry/deploy_registry_existing_clusters.adoc]: Should "Service..." above have caps? Goes for the lines below as well, as well.

**Comment by mburke5678** [comment]
@bfallonf @zhaozhanqi @rjhowe PTAL at the changes I made based on Brice's comments.

**Comment by bfallonf** [comment]
@mburke5678 LGTM

**Comment by zhaozhanqi** [comment]
LGTM

**Comment by mburke5678** [comment]
Merged and CP's in https://github.com/openshift/openshift-docs/pull/8714

**Pull request merged**